test/RecipeBot.Domain.Test/Models/RecipeModelTest.cs

```diff
@@ -188,21 +211,17 @@ public void Model_with_image_url_and_empty_recipe_fields_returns_total_length_of
         Assert.Equal(expectedLength, totalLength);
     }
 
-    private static AuthorModel CreateAuthor()
+    private static AuthorModel CreateAuthor(Fixture fixture)
     {
-        const string authorImageUrl = "http://www.google.com";
-
-        var random = new Random(21);
-        var authorName = new string('o', random.Next(100));
-
-        return new AuthorModel(authorName, authorImageUrl);
+        return fixture.Build<AuthorModel>()
+                      .FromFactory<string>(author => new AuthorModel(author, imageUrl))
+                      .Create(); ;
     }
 
-    private static RecipeFieldModel CreateFields(int seed)
+    private static RecipeFieldModel CreateFields(Fixture fixture)
     {
-        var random = new Random(seed);
-        var fieldName = new string('+', random.Next(100));
-        var fieldData = new string('x', random.Next(100));
+        var fieldName = fixture.Create<string>();
+        var fieldData = fixture.Create<string>();
```

Comment:
Perhaps use a combination of registering how to create a field data and a CreateMany with the autofixture?